sdk/ruby/lib/chain/transaction_feed.rb

```diff
@@ -104,6 +109,7 @@ def delete(opts)
         nil
       end
 
+      # @param [QueryOpts || Hash] query
```

Comment:
We may want to make it parameterizable so it accepts `page_size`. According to this it does:

https://github.com/chain/chain/blob/main/docs/internal/api-spec.swagger.yml#L998